master/pkg/tasks/task_command.go

```diff
@@ -46,7 +46,9 @@ type GenericCommandSpec struct {
 
 	CommandID string
 
-	Config          model.CommandConfig
+	Config model.CommandConfig
+	// Deprecated: kept so we can still marshal to this.
+	// Please use command.CreateGeneric.modelDef instead.
 	UserFiles       archive.Archive
```

Comment:
this is only for command restore or something, right? and it never uses it, so we dont need line 103, we just need it here to not explode when pulling it from the db?

Reply:
103 as in right? https://github.com/determined-ai/determined/blob/3f9a980ec1eec651f60d07b69327adebf181e8e1/master/pkg/tasks/task_command.go#L103

Yeah I think this is where it will be eventually pulled from the database
https://github.com/determined-ai/determined/blob/3f9a980ec1eec651f60d07b69327adebf181e8e1/master/internal/command/models.go#L24